src/Stratis.Bitcoin.Features.Miner.Tests/PosMintingTest.cs

```diff
@@ -351,6 +352,100 @@ public void GetNetworkWeight_UsesLast73Blocks_CalculatedNetworkWeightUsingLatest
 
             Assert.Equal(4607763.9659653762, weight);
         }
+        
+        [Fact]
+        public void CoinstakeAge_BeforeActivation_Testnet()
+        {
+            bool suitable = this.WasUtxoSelectedForStaking(Network.StratisTest, 1000, 1000 - 8); // utxo depth is 9, mining block at 10
+            Assert.True(suitable);
+
+            bool suitable2 = this.WasUtxoSelectedForStaking(Network.StratisTest, 1000, 1000 - 7); // utxo depth is 8, mining block at 9
```

Comment:
2? seriously? 😃

Reply:
Yeah. Sounded better than moving it to a separated test

Reply:
Is it just testing the either side of the activation point?
I'd inline it to the Assert.True/False and not have the variable at all then possibly.

Reply:
Ok, fixed